internal/ingress/controller/controller.go

```diff
@@ -1331,3 +1331,19 @@ func getRemovedIngresses(rucfg, newcfg *ingress.Configuration) []string {
 
 	return oldIngresses.Difference(newIngresses).List()
 }
+
+// isCanary returns whether an Ingress has the canary annotation
+// or contains 'canary' in its name.
+func isCanary(ing *ingress.Ingress) bool {
+
+	if ing.ParsedAnnotations.Canary.Enabled {
+		return true
+	}
+
+	if strings.Contains(ing.Name, "canary") {
+		klog.Warningf("enforce canary on Ingress %v/%v because of it's name", ing.GetNamespace(), ing.GetName())
```

Comment:
Did you decide not to update this message?

Reply:
hum, I did update it 😕

Reply:
did not push ... that's good now